src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started work");
```

Comment:
The message "Excavator started work" is not very informative. Consider specifying that it is the Excavator machine that is working, to make it clear and informative as per the checklist.